test-fixtures/gradle-multi-project/.vscode/settings.json

```diff
@@ -0,0 +1,79 @@
+{
+  "editor.formatOnSave": false,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.spotlessGradle": true
+  },
+  "java.format.enabled": false,
+  "typescript.validate.enable": false,
+  "typescript.format.enable": false,
+  "javascript.validate.enable": false,
+  "javascript.format.enable": false,
+  "spotlessGradle.diagnostics.enable": false,
+  "spotlessGradle.format.enable": true,
+  "[java]": {
+    "editor.defaultFormatter": "richardwillis.vscode-spotless-gradle",
+    "spotlessGradle.diagnostics.enable": true
+  },
+  "[kotlinscript]": {
```

Comment:
You can remove everything down from here as we are not testing for these settings, only `java`. You can also remove all the `typescript.*` and `javascript.*` settings above.